src/js/components/Page.js

```diff
@@ -192,6 +210,7 @@ var Page = React.createClass({
 
     return (
       <div className={classSet}>
+        {this.getTopBanner()}
```

Comment:
Maybe we should put `<MountService.Mount type="Page:TopBanner" />` right here and get rid of the `getTopBanner` entirely?